Package@swift-5.6.swift

```diff
@@ -0,0 +1,146 @@
+// swift-tools-version:5.6
```

Comment:
Can we use the style that the rest of the NIO repos use, where we have a Package@swift-5.4.swift and a Package@swift-5.5.swift? This lets us age out old package support really easily.